<?xml version="1.0"?>
<launch>
  <arg name="data_dir" default="$(env HOME)/region_detection_data/split_images/rectangle"/>
  <node name="detect_regions_demo" pkg="region_detection_roscpp_demo" type="detect_regions_demo" output="screen">
  	<param name="region_detection_cfg_file" 
  		value="$(find region_detection_roscpp_demo)/config/region_detect_mult_imgs.yaml"/>
  	<param name="data_dir" value="$(arg data_dir)"/>
  	<rosparam command="load" ns="data" file="$(find region_detection_roscpp_demo)/config/data_multiset.yaml"/>
  </node>
  <node pkg="tf" type="static_transform_publisher" name="results_frame" args="0 0 1.2 0 0 3.14 world results_frame 100" />
  <node name="rviz" type="rviz" pkg="rviz" args="-d $(find region_detection_roscpp_demo)/config/demo.rviz"/>
</launch>